tyk-pro/values.yaml

```diff
@@ -76,6 +78,21 @@ mongo:
   # Default value: false
   useSSL: false
 
+postgres:
+  # Default value: tyk-postgres-postgresql.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
It's a bit confusing that we write default values before the field name and also twice

Reply:
Agreed